tests/Trakx.CoinGecko.ApiClient.Tests/Integration/CoinGeckoClientTests.cs

```diff
@@ -146,6 +146,27 @@ public async Task GetMarketData_should_return_valid_data_when_passing_valid_id()
         }
     }
 
+    [Fact]
+    public async Task GetAllPricesForSymbols_gets_prices_for_expected_symbols()
```

Comment:
`GetAllPricesForSymbols` is a little too complex for a bunch of unit tests, so adding an integration test that covers all scenarios - ranked and unranked coins - is faster and more robust.